code/src/cljs/sixsq/nuvla/ui/edge_detail/subs.cljs

```diff
@@ -43,6 +44,27 @@
     (get nuvlabox-peripherals id)))
 
 
+(reg-sub
+  ::nuvlabox-events
+  (fn [db]
+    (::spec/nuvlabox-events db)))
+
+(reg-sub
+  ::elements-per-page
+  (fn [db]
+    (::spec/elements-per-page db)))
+
+(reg-sub
```

Comment:
These specs, elements-per-page and page are specific for events tab. When we will add a tab for deployments, we will have to rename page and elements-per-page in specs and try to not break paging of events tab.